contracts/strategies/LPOptimiserStrategy.sol

```diff
@@ -43,11 +57,133 @@ contract LPOptimiserStrategy is DefaultAccessControl, ILpCallback {
         _erc20Vault = erc20vault_;
         _vault = vault_;
         _marginEngine = IMarginEngine(vault_.marginEngine());
+        _periphery = IPeriphery(vault_.periphery());
+        _vamm = IVAMM(vault_.vamm());
         _currentPosition = vault_.currentPosition();
         _tokens = vault_.vaultTokens();
         _pullExistentials = vault_.pullExistentials();
```

Comment:
We'll need an event here that captures strategy inception.

Reply:
Now there is an event called StrategyDeployment